src/CommandLine/CommandRunner.cs

```diff
@@ -13,9 +13,9 @@ public static async Task Run(CommandOption connectionString, Func<ManagementClie
 
             var client = new ManagementClient(connectionStringToUse);
 
-            await func(client);
+            await func(client).ConfigureAwait(false);
```

Comment:
We need to revert all the changes to the CommandLine project.

Reply:
I thought I dropped that commit but I didn't. Doing it now.